dotnet-interactive.Tests/CommandLine/CommandLineParserTests.cs

```diff
@@ -102,7 +103,10 @@ public void jupyter_install_command_parses_path_option()
 
             var option = result.CommandResult.OptionResult("--path");
 
+            using var scope = new AssertionScope();
+
             option.Should().NotBeNull();
+            result.FindResultFor(option.Option).GetValueOrDefault<DirectoryInfo>().FullName.Should().BeEquivalentTo(_kernelSpecInstallPath.FullName);
```

Comment:
For a `string` you can use `Be` instead of `BeEquivalentTo`

Reply:
Done